### DISCUSSION TITLE
Getting an array from 1 to number of columns from csv output

### DISCUSSION THREAD

**Discussion by zillurbmb51** [nextflow-io/nextflow#3381] [Q&A]
One of my process gives output of one csv file. I want to create an array channel from 1 to number of columns. For example:
My output

    my_out_ch.view() -> test.csv

Assume, test.csv has 11 columns. Now I want to create a channel which gives me:

    1,2,3,4,5,6,7,8,910,11
How could I get this? I have tried with `splitText` operator as below without luck:

    my_out_ch.splitText(by:1,limit:1)
But it only gives me the columns names. There is a parameter `elem`, I am not sure if `elem` could give me the array and also not sure how to use it. Any help?

**Comment by jemunro** [comment]
Try:
```nextflow
my_out_ch.splitCsv(skip:1)
```
see https://www.nextflow.io/docs/latest/operator.html#splitcsv
